metron-platform/metron-elasticsearch/src/test/java/org/apache/metron/elasticsearch/integration/components/ElasticSearchComponent.java

```diff
@@ -178,8 +195,11 @@ public boolean hasIndex(String indexName) {
 
     @Override
     public void stop() {
-        node.close();
+        try {
+            node.close();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
We either want to throw up the exception or at least log it with a logger, rather than stdout.